### PR TITLE
Add time_signal command parsing

### PR DESCRIPTION
**Pull request by davemevans**
Comments welcome.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -130,6 +130,9 @@ pub enum SpliceCommand {
         reserved: u8,
         splice_detail: SpliceInsert,
     },
+    TimeSignal {
+        splice_time: SpliceTime
```
Comment (davemevans) [inline src/lib.rs]: This works, but I wonder if it should be `Timed` - I initially mistook that `SpliceTime` was analogous to `splice_time`, which it doesn't appeat to be.

Comment (davemevans) [reply src/lib.rs]: No, it doesn't work like that. I have much Rust to learn.

**Comment by dholroyd** [comment]
:confetti_ball:

**Comment by dholroyd** [comment]
Should have thought to ask prior to merge -- is that the full syntax of `time_signal()` now supported -- i.e. can I tick that box in README.md now?

**Comment by davemevans** [comment]
Yes, I believe so 👍

**Pull request merged**